controlplane/eks/api/v1beta1/conversion.go

```diff
@@ -31,6 +32,13 @@ func (r *AWSManagedControlPlane) ConvertTo(dstRaw conversion.Hub) error {
 	if err := Convert_v1beta1_AWSManagedControlPlane_To_v1beta2_AWSManagedControlPlane(r, dst, nil); err != nil {
 		return err
 	}
+	
```

Comment:
```suggestion
	if r.Spec.DisableVPCCNI != dst.Spec.VpcCni.Disable {
		dst.Spec.VpcCni.Disable = r.Spec.DisableVPCCNI
	}
	
	
```

Reply:
Why if check? What if we directly assign ?

Reply:
Good point. No reason to I guess.

Reply:
So this change is already present after `Unmarshal`.

Reply:
No, that's not converting from `DisableVPCCNI` => `VpcCni.Disable`

Reply:
Done

Reply:
It still the same. Am I missing something or has GitHub glitched?

Reply:
Oh i made this change, didn't reflect, I will add again